README.rst

```diff
@@ -215,11 +215,12 @@ Each test function should have the ``case`` fixture, that represents a single te
 
 Important ``Case`` attributes:
 
-- ``method`` - HTTP method
-- ``formatted_path`` - full endpoint path
-- ``headers`` - HTTP headers
-- ``query`` - query parameters
-- ``body`` - request body
+- ``input_type`` - type of data in the case, it can be valid or invalid;
```

Comment:
What will the default be? Would like to have a way to generate both valid and invalid data in the same run of tests.